playbooks/roles/custom_certificates/tasks/issue.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I don't think you need this file, and it's actually what's causing the role to not run anything. I'm unsure why, but removing it seemed to fix the issue we were talking about yesterday.